cime_config/allactive/config_compsets.xml

```diff
@@ -421,6 +421,11 @@
   <lname>2000_DATM%JRA-1p5_SLND_MPASSI_MPASO%DATMFORCED_DROF%JRA-1p5_MALI%SIA_SWAV</lname>
 </compset>
 
+<compset>
+  <alias>MPAS_FOLISIO_JRA1p5</alias>
+  <lname>2000_DATM%JRA-1p5_SLND_MPASSI_MPASO%DATMFORCED_DROF%JRA-1p5_MALI_SWAV</lname>
+</compset>
+
```

Comment:
And should it be `TL319_oQU240wLI_ais8to30`?

Reply:
Good questions. 

I'm inclined for now to leave it using the SIA solver so that it can be run on all machine and compilers, but I'm open to other opinions.  

Regarding switching to the ais8to30 MALI mesh, I think either is fine.  If we are considering ais8to30 as an improved low res testing mesh, then, yes, I suppose it would make sense to make that transition.

Reply:
We discussed this on Slack and decided yes to `ais8to30`, no to `FOLISIO`.